submissions/pashapushak1996/html-css-popup/scss/_layout.scss

```diff
@@ -0,0 +1,200 @@
+@import "base";
+
+body {
+  background-color: $white;
+  color: $grey-2;
+}
+
+.wrapper {
+  min-height: 100%;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding: 2rem;
+  &__list {
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+  }
+
+  &__link {
+    color: $grey-2;
+    text-decoration: none;
+    &:focus {
+      outline: none;
+      border-radius: 1px;
+      box-shadow: 0 0 0 2px $blue-1;
+    }
+  }
+
+  &__link_notify {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 2.5rem;
+    height: 2.5rem;
+  }
+
+  &__profile {
```

Comment:
this is a copy what we have above except one line, try do not duplicate your code.